src/fiskaltrust.Launcher/ProcessHost/ProcessHostMonarch.cs

```diff
@@ -40,10 +40,10 @@ public ProcessHostMonarch(ILogger<ProcessHostMonarch> logger, LauncherConfigurat
                 "--launcher-configuration", $"\"{Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(launcherConfiguration.Serialize()))}\"",
             });
 
-            if (Debugger.IsAttached)
-            {
-                _process.StartInfo.Arguments += " --debugging";
-            }
+            // if (Debugger.IsAttached)
```

Comment:
I want to actually have this commented out per default because it leads to unexpected behavior if you don't know about these lines of code (the launcher waiting until you attach a debugging session).